packages/field-base/src/utils.d.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Please add license header to this file for consistency with the other files in this package.